templates/pages/cart.html

```diff
@@ -24,6 +24,8 @@
                 {{> components/cart/totals}}
             </div>
 
+            {{{region name="banner_below_price"}}}
```

Comment:
I would recommend being more specific with the name on a cart page (as it is not a generic region like in the header), for instance, `cartpage_banner_below_totals`

Reply:
Okay. Thank you!